.dockerignore

```diff
@@ -0,0 +1,4 @@
+Dockerfile
+.gitignore
+.git
```

Comment:
Is the version info in the footer hardcoded rather than determined from git context?

Reply:
Do we have a version on the footer of the public site?

Reply:
![image](https://user-images.githubusercontent.com/5818258/103042256-0099b500-4547-11eb-9561-6005dbd58dec.png)
The footer just has a link to the GH repo itself, no like commit hash or anything

Reply:
That's what I thought. We should be good to ignore it.